lib/ex_sdp/attribute/rtcp_feedback.ex

```diff
@@ -0,0 +1,81 @@
+defmodule ExSDP.Attribute.RTCPFeedback do
+  @moduledoc """
+  This module describes what kind(s) of RTCP Feedback will be used (RFC 4585).
+
+  The example lines that are parsed by this module are (`a=rtcp-fb:` prefix will be handled by `ExSDP.Attribute`)
+
+      a=rtcp-fb:* nack
+      a=rtcp-fb:96 goog-remb
+      a=rtcp-fb:96 transport-cc
+      a=rtcp-fb:96 ccm fir
+      a=rtcp-fb:96 nack
+      a=rtcp-fb:96 nack pli
+
+  """
+
+  alias ExSDP.Attribute.RTPMapping
+
+  @type t :: %__MODULE__{
+          pt: RTPMapping.payload_type_t() | :all,
+          feedback_type: feedback_type_t()
+        }
+
+  @enforce_keys [:pt, :feedback_type]
+  defstruct @enforce_keys
+
+  @typedoc """
+  Key that can be used for searching this attribute using `ExSDP.Media.get_attribute/2`.
+  """
+  @type attr_key :: :rtcp_feedback
+
+  @type feedback_type_t() :: :nack | :fir | :pli | :twcc | :remb | binary()
+
+  @fb_type_mapping %{
+    "nack" => :nack,
+    "ccm fir" => :fir,
+    "nack pli" => :pli,
+    "transport-cc" => :twcc,
+    "goog-remb" => :remb
+  }
+
+  @reverse_fb_type_mapping Map.new(@fb_type_mapping, fn {k, v} -> {v, k} end)
+
+  @spec parse(binary()) :: {:ok, t()}
+  def parse(rtcp_fb) do
+    [pt_string, rest] = String.split(rtcp_fb, " ", parts: 2)
+
+    pt =
+      case pt_string do
+        "*" -> :all
+        _pt -> String.to_integer(pt_string)
+      end
+
+    {:ok, %__MODULE__{pt: pt, feedback_type: parse_feedback_type(rest)}}
+  end
+
+  defp parse_feedback_type(fb_type_string) do
+    Map.get(@fb_type_mapping, fb_type_string, fb_type_string)
```

Comment:
Why we don't use `Map.fetch` here?

Reply:
To have a fallback to string. With `fetch` we'd have to do
```suggestion
    case Map.fetch(@fb_type_mapping, fb_type_string) do
      {:ok, fb_type} -> fb_type
      :error -> fb_type_string
    end
```